.travis.yml

```diff
@@ -82,8 +82,7 @@ script:
     - if [ $TASK_TESTS == 1 ] && [ $TASK_TESTS_COVERAGE == 0 ]; then vendor/bin/phpunit --verbose; fi
     - if [ $TASK_TESTS == 1 ] && [ $TASK_TESTS_COVERAGE == 1 ]; then phpdbg -qrr vendor/bin/phpunit --verbose --coverage-clover build/logs/clover.xml; fi
 
-    - if [ $TASK_SCA == 1 ]; then git checkout . -q; fi
-
+    - if [ $TASK_CS == 1 ]; then git checkout . -q; fi
```

Comment:
why this change? if no SCA was performed, there is no files to checkout, isn't it ?